### ISSUE TITLE
Client deletes remote files without asking

### ISSUE THREAD

**Issue by voroyam** [owncloud/client#7109]
### Use Case

* Install the sync client
* Select the Virtual Files Option
* Decide you don't want that Option
* Look for a way to have your files physically on your storage again
* Remove the Checkbox - no effect
* Remove and Install the Client - no effect
* Remove the Client, remove all the files in your sync dir, install the Clinet 

### Expected behavior
having the client sync the files from your remote location to your local storage

### Actual behavior
client deletes all the files on your remote storage

OS: Windows 10, latest updates
Client Version: 2.5.4

**Comment by jnweiger** [comment]
The last step 'Remove the Client, remove all the files in your sync dir, install the Clinet' probably has the same effect as 'remove all the files in your sync dir while client is running'
That operation is dangerous. It instructs the server to delete files. 

A safe version would be 'stop the client, remove the sync dir (including the database), start the client'. Then the newly started client looses its history, and would not delete files.

**Comment by ckamm** [comment]
I consider this a bug because there's a safeguard for this case. This warning message should be shown:

> All the files in your local sync folder 'owncloud-admin' were deleted. These deletes will be synchronized with your server, making such files unavailable unless restored.
Are you sure you want to sync those actions with the server?
If this was an accident and you decide to keep your files, they will be re-synced from the server.

That's what I get when I attempt to reproduce.

@voroyam Do you happen to have "sync hidden files" enabled? Maybe there's a hidden file in your sync folder that you didn't delete - and then the safeguard doesn't trigger because you didn't delete *all* files.

**Comment by hodyroff** [comment]
If its not reproducable its not a bug. Perfect usecase for the complex question.
@voroyam can we close?

**Comment by jnweiger** [comment]
Users generally do not expect any real file removals when they remove placeholders. 

I have seen more cases like that. This misconception ist the issue we should address.

**Comment by voroyam** [comment]
I can try to reproduce this again this week and post my findings. I am pretty sure that this was not fixed because it was not considered a bug and we had no customers reporting this. So no priority.

If I recall correctly I tried to disable virtual file system, it did not work, I removed the client and installed is again - the config files weren't removed and I got virtual file system again. 

Then I simply removed everything, sync folder and the ownCloud installation folder on Windows to be sure, then installed the client again, and then all my remote files were removed with no warning because the client looked in the sync folder, saw that it's empty, and assumed that I removed all the files. Luckily I had them in my trash bin, but it's quite a shock if your files were removed.
